controllers/api/posts.js

```diff
@@ -81,4 +83,60 @@ async function destroy(req, res, next) {
     } catch (error) {
         res.status(400).json({ msg: error.message })
     }
-}
\ No newline at end of file
+}
+
+// Like Post
+async function likePost(req, res, next) {
+    try {
+        const postId = req.params.id
```

Comment:
`postId` variable not necessary because you only use it once. you might want to save `req.user` though, because you use that a lot here.